src/v2i-hub/PortDrayagePlugin/src/PortDrayagePlugin.cpp

```diff
@@ -223,27 +227,28 @@ CARMAFreightPlugin::PortDrayage_Object CARMAFreightPlugin::retrieveNextAction( s
 			sql::ResultSet *cur_action = next_action->executeQuery();
 			// Create PortDrayage_Object
 			PortDrayage_Object *rtn = new PortDrayage_Object();
-			if ( cur_action->first() )  {
```

Comment:
This check was deleted, but shouldn't we still be checking whether an sql::ResultSet exists for the next action id?